src/utils/swapErrorToUserReadableMessage.tsx

```diff
@@ -25,14 +25,18 @@ export function swapErrorToUserReadableMessage(error: any): string {
   // The 4001 error code doesn't capture the case where users reject a transaction for all wallets,
   // so we need to parse the reason for these special cases:
   if (
+    // ethers v5.7.0 wrapped error
+    error?.code === 'ACTION_REJECTED' ||
```

Comment:
you could import this from ethers directly :D its `Logger.errors.ACTION_REJECTED` sorry i didnt have it on my branch!

Reply:
i think we need the ethers upgrade to 5.7.0 first, not seeing it ACTION_REJECTED in Logger.errors right now :O

Reply:
very true